### ISSUE TITLE
a bytes-like object is required, not 'str'

### ISSUE THREAD

**Issue by alexanderahlsen** [alexal1/Insomniac#429]
Hi,

Everything seems ti have been working, and now I get this error:
```
Engine v3.8.1

[04/01 14:40:55] Your activation code is confirmed, welcome!
a bytes-like object is required, not 'str'
```

Anyone know why? I'm running the correct IG version.

**Comment by alexal1** [comment]
Try to run with `--debug` and attach the full log here; right now not enough information

**Comment by alexanderahlsen** [comment]
> Try to run with `--debug` and attach the full log here; right now not enough information
Even adding --debug in the console or in the config file, I still just get the same in the console:
Engine v3.8.1

[04/02 10:01:48] Your activation code is confirmed, welcome!
a bytes-like object is required, not 'str'

Seems like it doesn't even get starter after checking the activation code.

**Comment by alexal1** [comment]
Fixed in v3.8.2
